.github/actions/setup-job/action.yml

```diff
@@ -50,6 +54,18 @@ runs:
         flutter-version: ${{ env.FLUTTER_VERSION }}
         cache: true
 
+    - name: Install Android NDK
+      if: inputs.android != 'false'
+      uses: nttld/setup-ndk@4a541f506ff27d545f89e860b7cdc86efc3ca496 # HEAD 2022-02-15
+      with:
+        ndk-version: ${{ env.ANDROID_NDK_VERSION }}
```

Comment:
We shouldn't need that. The Ubuntu runner should have ndk installed [ref](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#environment-variables-3)